database/certificate.go

```diff
@@ -452,11 +467,11 @@ type Scannable interface {
 func (db *DB) scanCert(row Scannable) (certificate.Certificate, error) {
 	cert := certificate.Certificate{}
 
-	var crl_dist_points, extkeyusage, keyusage, subaltname, policies, issuer, subject, key []byte
+	var crl_dist_points, extkeyusage, extkeyusageOID, keyusage, subaltname, policies, issuer, subject, key []byte
```

Comment:
Same nit as above

Reply:
Yeah, I'm aware of Go's convention. This was so named to stay inline with the surrounding names that already existed ("when in Rome..."). I can still change these if you'd like, or do a general refactor of the names in the package for a little bit of spring cleaning.